src/pages/pass-finalisation/pass-finalisation.html

```diff
@@ -26,6 +26,14 @@ <h3 id="candidate-driver-number">{{pageState.candidateDriverNumber$ | async}}</h
           <ion-col col-36>
             <label>Provisional license received</label>
           </ion-col>
+          <ion-col col-23>
+            <input type="radio" name="license" id="license-received" value="yes" [(ngModel)]="pageState.provisionalLicenseValue" class="gds-radio-button" (click)="provisionalLicenseReceived()">
```

Comment:
Instead of using `[(ngModel)]`, could you just do something like `[checked]="pageState.provisionalLicenceProvided$ | async"`?